test/e2e/spinup_test.go

```diff
@@ -58,6 +59,7 @@ func spinup(t testing.TB, cfg config) (close func()) {
 			"--cluster.peers", "127.0.0.1:19591",
 			"--log.level", "debug",
 		))
+		time.Sleep(200 * time.Millisecond)
```

Comment:
why not putting more peers?

Reply:
Yea, it's a hot fix. I started building a slice of all peers – then realized it need more work to pass them all as flags :P